app/views/sessions/welcome.html.erb

```diff
@@ -0,0 +1,30 @@
+<% if validate %>
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo01" aria-controls="navbarTogglerDemo01" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
+      <div class="navbar-brand"> <%= link_to "Home", '/welcome' %></div>
+      <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
+        
+      </ul>
+      <span class="form-inline my-2 my-lg-0">
+        <span class="btn btn-outline-success my-2 my-sm-0 mr-2"><%= link_to 'Log In', login_path %> </span>
+        <span class="btn btn-outline-success my-2 my-sm-0"><%= link_to 'Sign Up', new_user_path %></span> 
+      </span>
+    </div>
+  </nav>
+<% end %>
+<h1 class="text-center">Welcome to Private Event App</h1>
+  <% if current_user %>
```

Comment:
- [ ] There should be no logic in the views. Please encapsulate all of the view-related logic in your view helpers: **(app/helpers/*_helper.rb)**